Makefile

```diff
@@ -49,27 +44,12 @@ test:
 
 docker-build:
 	docker build --rm -t ${IMAGE} rootfs
+	docker tag -f ${IMAGE} ${MUTABLE_IMAGE}
 
 # Push to a registry that Kubernetes can access.
 docker-push:
 	docker push ${IMAGE}
 
-# Deploy is a Kubernetes-oriented target
-deploy: kube-service kube-rc
-
-# Some things, like services, have to be deployed before pods. This is an
-# example target. Others could perhaps include kube-secret, kube-volume, etc.
-kube-service:
-	kubectl create -f ${SVC}
-
-# When possible, we deploy with RCs.
-kube-rc:
-	kubectl create -f ${RC}
-
-kube-clean:
-	kubectl delete rc deis-${SHORT_NAME}
-	kubectl delete svc deis-${SHORT_NAME}
```

Comment:
:+1: on deleting these targets here, but @sgoings or @helgi can you create an issue in this repo for porting `make deploy` into here?